RamsaySt.roboFontExt/lib/ramsaySt.py

```diff
@@ -92,6 +97,15 @@ def glyphEditorDidMouseDown(self, info):
                 if self.rightGlyph.pointInside((x - glyph.width, y)):
                     self.getGlyphEditor().setGlyph(self.rightGlyph)
 
+    def adjunctGlyphDidChange(self, info):
+        self.setGlyph(info["glyph"])
```

Comment:
this must be `self.getGlyphEditor().getGlyph()` 

the `info["glyph"]` will the adjunct glyph here, not the glyph in the glyph editor

Reply:
Aha – I can change this of course. One question: since this method is mostly relevant if the neighbor is the same as the center glyph, would this change make a difference?

Reply:
the adjunct glyph could change somewhere else, in a second open glyph editor.